### PR TITLE
Add forecast for all the cantons and plot of cases and hosp

### PR DESCRIPTION
**Pull request by fccoelho**

### CODE REVIEW

File: for_other_cantons.py [hunk 1]
```diff
@@ -0,0 +1,371 @@
...
+import streamlit as st
+import plotly.graph_objects as go
+from forecast import download_button
+from get_data import get_canton_data  
+from sqlalchemy import create_engine
+engine = create_engine("postgresql://epigraph:epigraph@localhost:5432/epigraphhub")
+
+
+
+#@st.cache
```
Comment (fccoelho) [inline for_other_cantons.py]: @eduardocorrearaujo  We can turn on caching on some of these functions

File: for_other_cantons.py [hunk 2]
```diff
@@ -0,0 +1,371 @@
...
+def get_curve(curve):
+    '''
+    Function to get the cases and hosp data in the database
+    '''
+    df = pd.read_sql_table(f'foph_{curve}', engine, schema = 'switzerland', index_col = 'datum', columns=[ 'geoRegion', 'entries'])
+    df.index = pd.to_datetime(df.index)
+    
+    return df
+
+def plot_cases(canton):
```
Comment (fccoelho) [inline for_other_cantons.py]: There is a function with the same name in forecast.py. we should eliminate code duplication, they just make code maintenance much harder.

File: for_other_cantons.py [hunk 3]
```diff
@@ -0,0 +1,371 @@
...
+    
+    # computing the rolling average 
+    m_movel = df.rolling(7).mean().dropna()
+    
+    fig = go.Figure()
+        
+    title = f"New Cases - {canton}"
+    
+    fig.update_layout(width=900, height=500, title={
+                'text': title,
```
Comment (fccoelho) [inline for_other_cantons.py:57]: @eduardocorrearaujo we could leave only canton name in the title of figures, the description of the data is already in the `yaxis_title`

**Review by fccoelho — COMMENTED** [review]
@eduardocorrearaujo just some minor adjustments and we can merge it.

File: for_other_cantons.py (hunk 1 above)
Comment (eduardocorrearaujo) [reply for_other_cantons.py]: When I try to turn on this function I receive this message:

**AttributeError: 'QueuePool' object has no attribute '_pool'**

Comment (fccoelho) [reply for_other_cantons.py]: This is due to an incompatibility between st.cache and Sqlalchemy. I have replaced it by cache tools

Comment (eduardocorrearaujo) [reply for_other_cantons.py]: Do you have an example of how to use these cache tools? Is It using the own streamlit st.chace function or other packages?

**Pull request merged**